packages/config-loader/src/loader.ts

```diff
@@ -14,22 +14,25 @@
  * limitations under the License.
  */
 
-import { AppConfig } from './types';
 import fs from 'fs-extra';
 import yaml from 'yaml';
-import { paths } from '../paths';
-
-type LoadConfigOptions = {
-  // Config path, defaults to app-config.yaml in project root
-  configPath?: string;
-};
+import { resolve as resolvePath } from 'path';
+import { AppConfig } from '@backstage/config';
+import { findRootPath } from './paths';
+import { LoadConfigOptions } from './types';
 
 export async function loadConfig(
   options: LoadConfigOptions = {},
 ): Promise<AppConfig[]> {
   // TODO: We'll want this to be a bit more elaborate, probably adding configs for
   //       specific env, and maybe local config for plugins.
-  const { configPath = paths.resolveTargetRoot('app-config.yaml') } = options;
+  let { configPath } = options;
+  if (!configPath) {
+    configPath = resolvePath(
+      findRootPath(fs.realpathSync(process.cwd())),
```

Comment:
I was a bit surprised to find use of cwd here, not saying it's wrong. I just have been bitten by that before in cases where you trigger the running of a tool from a shell in a dir way outside of the path that it then actually targets.